lib/config/presets/index.spec.ts

```diff
@@ -16,10 +18,12 @@ import * as presets from '.';
 jest.mock('./npm');
 jest.mock('./github');
 jest.mock('./local');
+jest.mock('../../util/cache/package');
```

Comment:
```suggestion
jest.mock('../../util/cache/package', () => mockDeep());
```

test perf (jest won't load the file to create a mock if a mock implementation is provided)